pcsx2/Docs/PCSX2_FAQ.md

```diff
@@ -0,0 +1,186 @@
+No. PCSX2 does not use or require SLI or Crossfire.
+
+### How do I check if a game is playable?
+The PCSX2 website has a [compatibility list](https://pcsx2.net/compatibility-list.html) with every game that has been tested. 
+
+### Do PS1 (PSX) games work on PCSX2?
+
+#### If you want to enjoy your PS1 games without issues
+No, they will not work. Please use a proper PS1 emulator, such as Mednafen.
+
+#### If you are a tinkerer and like to break things
+PS1 games may work on PCSX2, but compatibility is very limited. Specific settings are often required to get a game to boot, and there are other universal problems including missing and pitch shifted audio, and severe FMV corruption. A [forum thread for PS1 compatibility](https://forums.pcsx2.net/Thread-PSX-Mode-Unofficial-Compatibility-List) is tracking some fixes and compatibility reports. 
+
+*While we encourage discussion about PS1 games and improving compatibility, bug reports are NOT being accepted for PS1 games. Any reports for PS1 games will be closed immediately as invalid.*
+
+### My game doesn't work like it did in an earlier PCSX2 version. Why?
+One change to the emulator may fix one game, but cause problems for another. If the issue is severe and not fixable with different settings, you can always revert back to the last known PCSX2 version to work, and report the build number that broke the game. [Development builds](https://pcsx2.net/download/development.html) are very helpful for finding the exact change that caused a regression. 
+
+### Why is PCSX2 slow?
+The PlayStation 2 is a complex console, and this complexity substantially raises the PC requirements to emulate it at full speed, accurately. [This forum thread](https://forums.pcsx2.net/Thread-Why-is-PCSX2-slow) helps explain some of the technical reasons behind it, and our current guidelines for PC requirements are listed above.
+
+### Where do I get a PS2 BIOS?
+You have to dump the BIOS files from your PlayStation 2 console. There is a BIOS dumper tool [available on the PCSX2 website](https://pcsx2.net/download/releases/tools.html) to do this. If you need a guide, check the bottom of this FAQ for links to BIOS guides.
+
+### Where do I get PS2 games?
+You can use games from your personal collection of discs, purchase them from local stores that resell old games, or look online at sites like eBay or Amazon.
+
+### Can I use PS2 discs directly with PCSX2?
+Yes. However, some games have speed problems, because unlike the PS2 which would constantly keep the disc spinning, PCSX2 will not do the same to your PC's disc drive. It is recommended to instead dump your games to ISO images. ImgBurn is a popular software to do this with, and we recommend [installing ImgBurn by using Ninite](https://ninite.com) to avoid any issues with the ImgBurn installer.
+
+### How do I play a game?
+After PCSX2 is set up:
+
+1) Click the CDVD menu
+    * If you are using an ISO image:
+
+        2) Check that "ISO" mode is selected.
```

Comment:
Github displays this markdown with a mix of letters and numbers:

1) Click the CDVD menu
    * If you are using an ISO image:

        2) Check that "ISO" mode is selected.
        3) Go to "ISO Selector", then Browse.
        4) Locate your ISO image.
        5) Click the System menu, then Boot ISO.